src/commands/moderation/Ban.ts

```diff
@@ -60,14 +60,13 @@ export default class Ban {
     }
 
     try {
+      await modEmbed(
+        { interaction, user, action: "Banned", duration, dm: true, dbAction: "BAN", expiresAt },
+        reason
+      );
       await guild.members.ban(user.id, { reason: reason ?? undefined });
```

Comment:
Pull it up, we need to try actually banning first before sending the embed

Reply:
errors when DMing + no mutual servers